src/components/job-overview/style.css

```diff
@@ -19,13 +19,13 @@
 .step-summary__content {
```

Comment:
For better responsiveness, add to `.steps-list__divider`:
    `grid-template-rows: max-content 1fr;`

Reply:
fixed